src/org-manager/containers/pending-pbas/pending-pbas.component.spec.ts

```diff
@@ -92,32 +92,36 @@ describe('PendingPBAsComponent', () => {
       fixture.detectChanges();
     });
 
-    it('should show a table with a single row that says there are none pending', () => {
-      const tds = utils.getDebugElements('td');
-      expect(tds.length).toEqual(1);
-      utils.checkText(tds[0], 'There are no pending PBAs');
+    it('should show there are no new PBA requests.', () => {
```

Comment:
Finicky amend - please remove the full stop at the end of the description for consistency.

Reply:
I see you've removed the full stop so resolving.